#best_config_

Graphical_interface: true
ShowTrajectory: false
Trajectory_length: 50
TimeBetweenFrames: 0.1
LoadModel: false
Path2Model: "/home/ros/catkin_ws/src/Tobot/open_tld_3d/models/my_face"
SavingFile: "/home/ros/catkin_ws/src/Tobot/open_tld_3d/models/my_face"
Tracking3D: true
queue_size : 100
